backend/benefit/applications/tests/test_ahjo_payload.py

```diff
@@ -71,6 +86,8 @@ def test_prepare_case_records(decided_application, settings):
     ]
 
     for attachment in application.attachments.all():
```

Comment:
`attachments.exclude(attachment_type=AttachmentType.PDF_SUMMARY)` instead of `attachments.all()`